docs/conf.py

```diff
@@ -294,8 +293,3 @@
     'boto': ('http://boto.cloudhackers.com/en/latest/', None),
     'boto3': ('http://boto3.readthedocs.io/en/latest/', None)
 }
-
-# Force the RTD theme for all builds
-import sphinx_rtd_theme
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
Fixes a doc build warning about `get_html_theme_path()` no longer being required to call.